packages/mds-compliance/api.ts

```diff
@@ -178,7 +178,7 @@ function api(app: express.Express): express.Express {
                             (deviceMapAcc: { [d: string]: Device }, device: Device) => {
                               /* istanbul ignore next */
                               if (!device) {
-                                throw new Error('device in DB but not in cache')
+                                return deviceMapAcc
                               }
                               return Object.assign(deviceMapAcc, { [device.device_id]: device })
```

Comment:
If you're not going to throw, you don't even need the `if` ... `Object.assign` will just put that device_id in the map as undefined.

Reply:
Ah, cool beans! I'll get rid of that `if` then.

Reply:
Alternately, just `return device ? Object.assign(deviceMapAcc, { [device.device_id]: device }) : deviceMapAcc`